### PR TITLE
brion 3.2.0 release

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

**Review by ferdonline — DISMISSED** [review]

File: var/spack/repos/builtin/packages/brion/package.py [hunk 1]
```diff
@@ -37,14 +37,10 @@ class Brion(CMakePackage):
     # TODO: bzip2 is a dependency of boost. Needed here because of linking
     # issue (libboost_iostreams.so.1.68.0 not finding libbz2.so)
     depends_on('bzip2')
-    depends_on('lunchbox')
-    depends_on('vmmlib')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/brion/package.py:41]: :clap: 

Although, wouldn't you want to pin that for certain versions? Or are we just dropping support outright?

**Comment by pramodk** [comment]
@ppodhajski : would you have some time to finalise this? lunchbox is causing [so much trouble](https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/hpc.spack-deployment.pull-requests/detail/hpc.spack-deployment.pull-requests/772/tests) these days for us because it's now dependency of NSE stack (via bluepy) and whenever we update neuron (with Intel), we often encounter build issues.

**Comment by ppodhajski** [comment]
Hi Pramod,
Brion PR has been merged only now: https://github.com/BlueBrain/Brion/pull/281. We will release it as 3.2.0 and I will finalize this PR

**Comment by pramodk** [comment]
Ok perfect! Thanks!

File: var/spack/repos/builtin/packages/brion/package.py (hunk 1 above)
Comment (ppodhajski) [reply var/spack/repos/builtin/packages/brion/package.py:41]: yes, actually we will keep previous release, so will keep them pinned to 3.1.0

**Comment by jorblancoa** [comment]
retest this please jenkins

**Comment by ppodhajski** [comment]
retest this please

**Comment by jorblancoa** [comment]
```
[2020-05-29T13:16:40.497Z] ## blacklisted or missing from upstream: pgi@19.4%gcc@4.8.5~amd~java+mpi+network+nvidia~single arch=linux-rhel7-x86_64

[2020-05-29T13:16:40.497Z] bash: ##: command not found
```

This issue in the CI is caused by a wrong pgi version. After merging/rebase with develop it should go away.

File: deploy/packages/bbp-packages.yaml [hunk 2]
```diff
@@ -39,7 +39,7 @@ packages:
       - py-morph-tool@0.1.14
       - py-morphio@2.3.4
       - py-neurodamus
-      - py-sonata-network-reduction@0.0.6 ^neuron%intel@19.0.4^py-ipython%gcc^lunchbox%gcc
+      - py-sonata-network-reduction@0.0.6 ^neuron%intel@19.0.4^py-ipython%gcc
```
Comment (matz-e) [inline deploy/packages/bbp-packages.yaml:42]: This line gives me the chills. At least the lunchbox is out \o/

**Comment by ppodhajski** [comment]
retest this please

**Comment by pramodk** [comment]
I see travis is just green but doesn't post the status here: https://travis-ci.org/github/BlueBrain/spack.

I re-triggered.

File: var/spack/repos/builtin/packages/brayns/package.py [hunk 3]
```diff
@@ -52,6 +52,8 @@ class Brayns(CMakePackage):
     depends_on('optix@5.0.1', when='+optix')
     depends_on('cuda', when='+optix')
 
+    patch('brion.patch', when='@develop')
+
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/brayns/package.py:56]: As discussed offline, this is something should be removed/changed in the future.

**Review by pramodk — APPROVED** [review]

**Pull request merged**